src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -4,693 +4,689 @@
+    type: bool
+    short-summary: Reset service principal for a managed cluster.
+  - name: --service-principal
+    type: string
+    short-summary: Service principal used for authentication to Azure APIs. This argument is required if `--reset-service-principal` is specified.
+  - name: --client-secret
+    type: string
+    short-summary: Secret associated with the service principal. This argument is required if `--service-principal` is specified.
+  - name: --reset-aad
+    type: string
+    short-summary: Reset Azure Active Directory configuration for a managed cluster.
+  - name: --aad-server-app-id
+    type: string
+    short-summary: The ID of an Azure Active Directory server application. This argument is required if `--reset-aad` is specified.
+  - name: --aad-server-app-secret
+    type: string
+    short-summary: The secret of an Azure Active Directory server application. This argument is required if `--reset-aad` is specified.
+  - name: --aad-client-app-id
+    type: string
+    short-summary: The ID of an Azure Active Directory client application. This argument is required if `--reset-aad` is specified.
+  - name: --aad-tenant-id
+    type: string
+    short-summary: Tenant ID associated with Azure Active Directory.
+
+examples:
+  - name: Update an existing Kubernetes cluster with new service principal.
+    text: az aks update-credentials -g MyResourceGroup -n MyManagedCluster --reset-service-principal --service-principal MyNewServicePrincipalID --service-principal MyNewServicePrincipalID --client-secret MyNewServicePrincipalSecret
+  - name: Update an existing Azure Active Directory Kubernetes cluster with new server app secret key.
+    text: az aks update-credentials -g MyResourceGroup -n MyManagedCluster --reset-aad --aad-server-app-id MyExistingAADServerAppID --aad-server-app-secret MyNewAADServerAppSecret --aad-client-app-id MyExistingAADClientAppID --aad-tenant-id MyAADTenantID
 """
 
 helps['aks upgrade'] = """
-    type: command
-    short-summary: Upgrade a managed Kubernetes cluster to a newer version.
-    long-summary: "Kubernetes will be unavailable during cluster upgrades."
-    parameters:
-        - name: --kubernetes-version -k
-          type: string
-          short-summary: Version of Kubernetes to upgrade the cluster to, such as "1.11.8" or "1.12.6".
-          populator-commands:
-          - "`az aks get-upgrades`"
+type: command
+short-summary: Upgrade a managed Kubernetes cluster to a newer version.
+long-summary: "Kubernetes will be unavailable during cluster upgrades."
+parameters:
+  - name: --kubernetes-version -k
+    type: string
+    short-summary: Version of Kubernetes to upgrade the cluster to, such as "1.7.12" or "1.8.7".
+    populator-commands:
+      - "`az aks get-upgrades`"
+examples:
+  - name: Upgrade a managed Kubernetes cluster to a newer version. (autogenerated)
+    text: az aks upgrade --kubernetes-version 1.10.9 --name MyManagedCluster --resource-group MyResourceGroup
```

Comment:
Maybe use a fresher version here, such as 1.12.6? Just because 1.10.9 will drop off the support matrix sooner and make this example stale.

Reply:
Great feedback for @glGarg and @JasonRShaver. Would the KB eventually pick up on these updated versions and is there a process for tweaking them on review like @mboersma suggests?